src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  $root:&;
```

Comment:
what are you improving, using this?

Reply:
I use $root in 2 loops:

.calendar {
  $root:&;
  
  &__day {
    @each $day, $margin in $start-week-day {
      #{$root}--start-day_#{$day}
      &:first-child {
        margin-left: $margin;
      }
    }

    @for $index from 28 to 32 {
      #{$root}--month-length_#{$index}
      &:nth-child(n + #{$index + 1}) {
        display: none;
      }
    }

Using $root:&; made it possible to use loops inside the parent element .calendar__day